generate is broken on the latest commit

If I go to HEAD~100 it works but on tip:
I get:

```
~/unified-runtime$ cmake --build build --target generate
Traceback (most recent call last):
  File "run.py", line 13, in <module>
    import parse_specs
  File "/unified-runtime/scripts/parse_specs.py", line 553, in <module>
    def _make_versions(d, max_ver : Version) -> list[Version]:
TypeError: 'type' object is not subscriptable
make[7]: *** [CMakeFiles/generate-code.dir/build.make:70: CMakeFiles/generate-code] Error 1
make[6]: *** [CMakeFiles/Makefile2:1742: CMakeFiles/generate-code.dir/all] Error 2
make[5]: *** [CMakeFiles/Makefile2:1749: CMakeFiles/generate-code.dir/rule] Error 2
make[4]: *** [Makefile:582: generate-code] Error 2
make[3]: *** [CMakeFiles/generate.dir/build.make:70: CMakeFiles/generate] Error 2
make[2]: *** [CMakeFiles/Makefile2:1768: CMakeFiles/generate.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:1775: CMakeFiles/generate.dir/rule] Error 2
make: *** [Makefile:595: generate] Error 2
```

--- a/scripts/parse_specs.py
+++ b/scripts/parse_specs.py
@@ -17,7 +17,7 @@ import copy
 from templates.helper import param_traits, type_traits, value_traits
 import ctypes
 import itertools
-from typing import Optional
+from typing import Dict, List, Optional
 from version import Version
 
 
@@ -504,7 +504,7 @@ def _validate_doc(f, d, tags, line_num, meta):
 """
     filters object by version
 """
-def _filter_version(d, max_ver: Version) -> Optional[dict]:
+def _filter_version(d, max_ver: Version) -> Optional[Dict]:
     ver = Version(d.get('version', default_version))
     if ver > max_ver:
         return None
@@ -550,7 +550,7 @@ def _filter_version(d, max_ver: Version) -> Optional[dict]:
 """
     creates docs per version
 """
-def _make_versions(d, max_ver : Version) -> list[Version]:
+def _make_versions(d, max_ver : Version) -> List[Version]:
     docs = []
     type = d['type']
     if 'function' == type or 'struct' == type:

